lib/runtime-c-api/doxyfile

```diff
@@ -118,7 +118,6 @@ WARN_LOGFILE           =
 #---------------------------------------------------------------------------
 INPUT                  = doc/index.md
 INPUT                 += wasmer.h
-INPUT                 += wasmer.hh
```

Comment:
The idea here is to avoid variants/members duplications in Doxygen for structs and enums.